api/writeAPIBlocking.go

```diff
@@ -69,28 +70,28 @@ func NewWriteAPIBlockingWithBatching(org string, bucket string, service http2.Se
 }
 
 func (w *writeAPIBlocking) EnableBatching() {
-	w.mu.Lock()
-	defer w.mu.Unlock()
-	if !w.batching {
-		w.batching = true
+	if atomic.LoadInt32(&w.batching) == 0 {
```

Comment:
Wouldn't be easier to enforce or document that EnableBatching can be called before any write is performed?

Reply:
It would be easier to document it. I don't think it will be easier to enforce it. The way how it is safe.